assets/manifest_template.toml

```diff
@@ -7,7 +7,8 @@
 # List packages you wish to install in your environment inside
 # the `[install]` section.
 [install]
-_FLOX_INIT_INSTALL
+# hello.pkg-path = "hello"
+# nodejs = { version = "^18.4.2", pkg-path = "nodejs_18" }
```

Comment:
suggestion: insert these comments programmatically. If we install an actual package, we don't need to provide a commented out example